samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -70,10 +71,19 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
     const [recognizer, setRecognizer] = React.useState<speechSdk.SpeechRecognizer>();
     const [isListening, setIsListening] = React.useState(false);
     const speechService = useSKSpeechService(process.env.REACT_APP_BACKEND_URI as string);
+    const chatRelay = useSKMultiUserChat(process.env.REACT_APP_BACKEND_URI as string);
     const [documentImporting, SetDocumentImporting] = React.useState(false);
     const documentImportService = useDocumentImportService(process.env.REACT_APP_BACKEND_URI as string);
     const documentFileRef = useRef<HTMLInputElement | null>(null);
 
+    React.useEffect(() => {
+        // This code will run once, when the component is mounted
+        console.log('SignalR setup called');
+        chatRelay.setupSignalRConnectionToChatHub();
+    // Disabling warning so that we can use empty dependency array to invoke this setup call just once 
```

Comment:
@Vivihung, @adrianwyatt  Is there a better way to run code only once on the frontend?

Reply:
@teresaqhoang / @TaoChenOSU  ?

Reply:
From a brief chat with vivian, disabling the warning and using an empty dependency array is the way to go, this should only call chatRelay.setupSignalRConnectionToChatHub(); once when the component is mounted. However there might be a bug somewhere thats making the component get mounted twice.